worlds/tracker/docs/map-integration.md

```diff
@@ -0,0 +1,91 @@
+# Map integration
+
+This document describes the requirements to add a map tab to UT. UT uses the [Poptracker](https://github.com/black-sliver/PopTracker/tree/master) specificiations for the content of the map tab.
+
+The intent of UT's map tab is to allow world devs to start making a poptracker pack before logic settles into a more stable configuration, however there's enough benefits to UT's integrated tab that it might be worth adding support even if the world already has a dedicated poptracker pack.
+
+## The Basics of adding a map page
+
+By defining a structure with the following fields, the world informs UT that a poptracker pack should be loaded
+
+```py
+from typing import ClassVar
+
+    tracker_world: ClassVar = {
```

Comment:
```suggestion
By defining a dict with the following fields, the world informs UT that a poptracker pack should be loaded

```py
from typing import ClassVar

class MyWorld(World):
    tracker_world: ClassVar = {
```
Should make it a little more clear this is a class attribute. ~ technically ~ any mapping would work instead of a dict specifically, but we should keep it simple